process/txsimulator/data/data.go

```diff
@@ -7,10 +7,6 @@ import (
 
 // SimulationResults is the data transfer object which will hold results for simulation a transaction's execution
 type SimulationResults struct {
```

Comment:
SimulationResultsWithVMOutput ? In order to distinguish between `transaction.SimulationResults` and `data.SimulationResults`

Reply:
done